server/UserRegistration.java

```diff
@@ -0,0 +1,467 @@
+        // set all methods to return values - created conversions for sortcode
+        // and account number to make it easier to push through to db.
+
+
+        newUserName nn = new newUserName();
+        String userFullname = nn.GetNameFromUser();
+
+
+        DateChecks dc = new DateChecks();
+        String userDoB = dc.GetDateFromUser();
+
+
+        UserId ui = new UserId();
+        String userUsername = ui.getNewUserId();
+
+
+        newUserAccountNumber acm = new newUserAccountNumber();
+        String userAccountNumber = acm.getNewUserAccountNumber();
+
+
+        newUserSortcode nusort = new newUserSortcode();
+        String userSortCode = nusort.getNewUserSortCode();
+
+
+        NewUserSecurity nus = new NewUserSecurity();
+
+        String userNewPass = nus.inputNewPassword();
+
+
+        NewUserBalance nub = new NewUserBalance();
+
+        Integer userNewBal = Integer.valueOf(nub.balance());
+
+
+        newUserSecretQuestion nsq = new newUserSecretQuestion();
+
+        String userNewSecretQuestion = nsq.secretQuestion();
+
+
+        checkNotARobot cnr = new checkNotARobot();
+
+        boolean NewUserNotARobot = cnr.checkHuman();
+
+        // All already scanned by scanner then passed as values to 'String addNewUser'
+        // The values are then passed to the db.
+
+
+        Connection conn = null;
+
+        try {
+            String url = "jdbc:sqlite:/Users/pmcdowell/Documents/nb/database/bank.sq3";
+
+            conn = DriverManager.getConnection(url);
+            // Will allow for the database connection.
+
+            String addNewUser = "INSERT INTO useroverview(name_user, dob_user, username_user, accountNumber_user, sortcode_user, password_user, balance_user, secretAns_user) VALUES (?, ?, ?, ?, ?, ?, ?, ?)";
+
+            // New values passed to db.
+            PreparedStatement updateuserdb = conn.prepareStatement(addNewUser);
+
+            updateuserdb.setString(1, userFullname);
+            updateuserdb.setString(2, userDoB);
+            updateuserdb.setString(3, userUsername);
+            updateuserdb.setString(4, userAccountNumber);
+            updateuserdb.setString(5, userSortCode);
+            updateuserdb.setString(6, userNewPass);
+            updateuserdb.setInt(7, userNewBal);
+            updateuserdb.setString(8, userNewSecretQuestion);
+
+
+            updateuserdb.executeUpdate();
+            conn.commit();
+            // The db will now push the output.
+
+
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
+        } finally {
+            try {
+                if (conn != null) {
+                    conn.close();
+                }
+            } catch (SQLException ex) {
+                System.out.println(ex.getMessage());
+            }
+        }
+
+
+        return null;
+    }
+
+
+    private static class newUserName {
+
+        Scanner sc = new Scanner(System.in);
+        String newName;
+
+        public String GetNameFromUser() {
```

Comment:
Nitpick: In Java, a convention is to name the method like this: `getNameFromUser`, this would be perfect in C#.

You can change this if you have time after fixing the other things :)